### PR TITLE
[10.0][ADD] quality_control_formula

### PR DESCRIPTION
**Pull request by SimoRubi**
New module to manage formulas in quality control.
Use case is the following: the value to be checked by the inspection is not what is inserted by the operator in Odoo but it is the result of some computation on that value (**input_value** in the module).

Such computation can also be performed based on the previous inspection lines of the same inspection.

### CODE REVIEW

**Comment by SimoRubi** [comment]
Travis to be fixed with #346

**Comment by SimoRubi** [comment]
@OCA/manufacturing-maintainers can someone please take a look? Thanks

**Comment by SimoRubi** [comment]
@OCA/manufacturing-maintainers can someone please take a look?
How can I help this PR to be merged?
Thanks

**Review by LoisRForgeFlow — APPROVED** [review]
Code review, LGTM :+1:

**Comment by LoisRForgeFlow** [comment]
@SimoRubi we still need one more review, maybe you can review other PR's and ask for a review here in exchange?

File: quality_control_formula/i18n/it.po
Comment (tafaRU) [inline quality_control_formula/i18n/it.po:53]: s/Formula Python formula/Formula Python

File: quality_control_formula/models/qc_inspection_line.py
Comment (tafaRU) [inline quality_control_formula/models/qc_inspection_line.py]: it would be cleaner to use `self.filtered(lambda l: l.formula)`

File: quality_control_formula/tests/test_quality_control_formula.py
Comment (tafaRU) [inline quality_control_formula/tests/test_quality_control_formula.py]: it would be better to get `3` as `int(inspection_line_d.quantitative_value)` likewise to when done for `inspection_line_c`.

**Review by tafaRU — CHANGES_REQUESTED** [review]
Both technical and functional review.
Just some nitpicking otherwise LGTM. Thanks!

**Comment by SimoRubi** [comment]
Thanks @tafaRU for the review!
I implemented your suggestions, I didn't exactly get what you meant about the `tests` file..
Please check if that's ok now

**Comment by tafaRU** [comment]
@SimoRubi thanks for the changes!

> I didn't exactly get what you meant about the tests file..

I meant to also calculate the result of the formula `A + B + int(input_value)` similarly to when done for the other one. It's non blocking comment anyway.

**Review by tafaRU — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
@SimoRubi are you going to add more changes or is it good to merge?

**Comment by SimoRubi** [comment]
Hi,
I don't see our customer willing to modify this feature in next future so this is good to merge, thanks!

**Comment by LoisRForgeFlow** [comment]
Let's do it then!

/ocabot merge

**Pull request merged**
